x-pack/plugins/cases/public/components/create/schema.tsx

```diff
@@ -10,7 +10,9 @@ import { FIELD_TYPES, fieldValidators, FormSchema } from '../../common/shared_im
 import * as i18n from './translations';
 
 import { OptionalFieldLabel } from './optional_field_label';
-const { emptyField } = fieldValidators;
+const { emptyField, maxLengthField } = fieldValidators;
+
+const MAX_TITLE_LENGTH = 64;
```

Comment:
I think you mean to import this from `x-pack/plugins/cases/common/constants.ts`?

Reply:
Yeap, missed that!